samples/zigbee/light_switch/README.rst

```diff
@@ -48,12 +48,14 @@ Once the light switch is successfully commissioned, it sends a broadcast message
 The light switch remembers the device network address from the first response.
 At this point, you can start using the buttons on the development kit to control the clusters on the newly found devices.
 
+Additionally, the Light switch sample powers down unused RAM sections to lower power consumption during sleep state.
+
 Sleepy End Device behavior
 ==========================
 
 The light switch supports the :ref:`zigbee_ug_sed` that enables the sleepy behavior for the end device, for a significant conservation of energy.
 
-To enable the sleepy behavior, press **Button 3** while the Light switch sample is booting.
+To enable the sleepy behavior, press **Button 3** while the Light switch sample is booting. This is required only when device is joining the network for the first time.
```

Comment:
```suggestion
To enable the sleepy behavior, press **Button 3** while the light switch sample is booting.
This is required only when device is joining the network for the first time.
After restarting the device, it will boot with the sleepy behavior enabled.
```